docs/admin/backup.rst

```diff
@@ -103,16 +103,15 @@ to create it, but it needs permissions to do so.
               - /home/weblate/data:/app/data
               - /home/weblate/borgbackup:/borgbackup
 
-    The directory where backups will be stored have to be owned by UID 1000,
-    otherwise Weblate will not be able to write the backups there.
+    To actually save backups, the directory backups are stored in has to be owned
+    by UID 1000.
 
 Remote backups
 ~~~~~~~~~~~~~~
 
-Remote backups using SSH are supported. The SSH server needs to have
-`BorgBackup`_ installed. Weblate connects to the server using SSH key, please
-make sure the Weblate SSH key is accepted by the server (see
-:ref:`weblate-ssh-key`).
+Install `BorgBackup`_ on the the SSH server set up somewhere else.
+You must make sure it accepts the Weblate SSH key Weblate uses to
+connect to it. (see :ref:`weblate-ssh-key`).
```

Comment:
Whole phrase soundsvery weird. I'd suggest something like this:

```
In order to create the remote backups you will have to install `BorgBackup`_
onto some another server accessible via SSH. You also will have to make sure
that it accepts the Weblate's client SSH key, i.e. the one it uses to connect
to other servers.
```

This one probably requires some more polishing too...

Reply:
included and polished